chord_metadata_service/restapi/api_views.py

```diff
@@ -138,6 +129,58 @@ def overview(_request):
     return Response(r)
 
 
+@api_view(["GET", "POST"])
+@permission_classes([OverrideOrSuperUserOnly])
+def search_overview(request):
+    """
+    get+post:
+    Overview statistics of a list of patients (associated with a search result)
+    - Parameter
+        - id: a list of patient ids
+    """
+    individual_id = request.GET.getlist("id") if request.method == "GET" else request.data.get("id", [])
```

Comment:
As it is an array that can contain multiple values, might be better to name individual_id as individual_ids

Reply:
Yes! Good suggestion. I'll add it in a subsequent PR.
Thanks for your review!